modules/apps/product-navigation/product-navigation-applications-menu-web/src/main/resources/META-INF/resources/js/ApplicationsMenu.js

```diff
@@ -346,6 +351,31 @@ const ApplicationsMenu = ({
 		onClose: () => setVisible(false),
 	});
 
+	useEventListener(
+		'keydown',
+		(event) => {
+			const isCMDPressed = Liferay.Browser.isMac()
+				? event.metaKey
+				: event.ctrlKey;
+
+			if (
+				isCMDPressed &&
+				event.shiftKey &&
+				event.key.toLowerCase() === 'm'
+			) {
+				event.preventDefault();
+
+				if (!visible) {
```

Comment:
Could we move this to the `handleTriggerButtonClick` method?

```
const handleTriggerButtonClick = () => {
	fetchCategories();
	setVisible(!visible);
};
```

I thing you don't need to check if `!visible` to call `fetchCategories` since that method is already taking care of not fetching twice.

Reply:
Looks good! I was a bit feared in change the `setVisible` of `handleTriggerButtonClick` from `false` to `!visible`.